public/apps/configuration/panels/role-view/role-view.tsx

```diff
@@ -22,33 +22,150 @@ import {
   EuiSpacer,
   EuiTabbedContent,
   EuiTitle,
+  EuiPageContent,
+  EuiText,
+  EuiLink,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPageBody,
+  EuiInMemoryTable,
+  EuiEmptyPrompt,
 } from '@elastic/eui';
 import { BreadcrumbsPageDependencies } from '../../../types';
 import { buildHashUrl } from '../../utils/url-builder';
 import { ResourceType, Action } from '../../types';
+import { getRoleMappingData, MappedUsersListing } from '../../utils/role-mapping-utils';
 
-const tabs = [
+interface RoleViewProps extends BreadcrumbsPageDependencies {
+  roleName: string;
+}
+
+const mappedUserColumns = [
   {
-    id: 'permissions',
-    name: 'Permissions',
-    disabled: false,
-    content: <Fragment>Permission working in progress</Fragment>,
+    field: 'user_type',
+    name: 'User type',
+    sortable: true,
   },
   {
-    id: 'users',
-    name: 'Users',
-    disabled: false,
-    content: <Fragment>Users working in progress</Fragment>,
+    field: 'user_name',
+    name: 'User',
+    sortable: true,
+    truncateText: true,
```

Comment:
Here looks different from mockup. It is array joined by string while mockup shows one user per line.

Reply:
I mapped as shown in following screenshot. 

![image](https://user-images.githubusercontent.com/63824432/88445042-1eff5700-cdd5-11ea-88ec-457e66fa71da.png)

User column is a string, not array of string.